env/types.go

```diff
@@ -204,7 +213,7 @@ var (
 
 func ParseBool(value, name string) (string, error) {
 	switch strings.ToLower(value) {
-	case "1", "t", "tr", "tru", "true", "on":
+	case "1", "t", "tr", "tru", "true", "on", "":
```

Comment:
My read of this section is that it looks like SQL session variables unrelated to client variables and using the standard libraries strconv.ParseBool vs the somewhat awkwardly named env package's [ParseBool](https://github.com/xo/usql/blob/master/env/types.go#L205)? Is there a code path through here that interacts with env.ParseBool I'm missing?

I looked at the other callers of env.ParseBool and it looked like the semantic change of "" == "on" would be ok as implied / consistent with QUIET for `\set` and `\pset` though it _is_ still a change:
- For `\timing` https://github.com/xo/usql/blob/master/metacmd/cmds.go#L418
- For boolean Pvars that control output formatting: https://github.com/xo/usql/blob/master/env/types.go#L320

I think you're right though that a semantics change should be avoided so I'll create a `ParseBoolEmptyOn` and use that as the parser func.

If someone later decided they want to change any semantics it would at least be easy to do so.